src/Types.hs

```diff
@@ -285,12 +285,21 @@ type instance Proto IdeaLike = ()
 data IdeaVote = IdeaVote
     { _ideaVoteMeta  :: MetaInfo IdeaVote
     , _ideaVoteValue :: IdeaVoteValue
+    , _ideaVoteVoter :: AUID User
```

Comment:
also comment on the fact that the delegation path can be lost over time.  (i don't see a problem with it now, but it should be made explicit.)